src/components/Kv/KvLightbox.vue

```diff
@@ -0,0 +1,168 @@
+<template>
+	<transition name="fade">
+		<div v-show="isShown" class="kv-lightbox">
+			<div class="row">
+				<div class="columns small=12">
+					<div class="lightbox-content" :class="removeContentBg">
+						<a @click.stop.prevent="closeLightbox" class="close-lightbox" aria-label="Close">
+							<kv-icon name="x" />
+						</a>
+						<slot name="title"></slot>
+						<slot>Lightbox content</slot>
+						<br>
+						<slot name="controls">
+							<kv-button v-if="showCloseButton" @click.native="closeLightbox">
+								Close
+							</kv-button>
+						</slot>
+					</div>
+				</div>
+			</div>
+		</div>
+	</transition>
+</template>
+
+<script>
+import KvIcon from '@/components/Kv/KvIcon';
+import KvButton from '@/components/Kv/KvButton';
+
+export default {
+	components: {
+		KvIcon,
+		KvButton
+	},
+	data() {
+		return {
+			isShown: false
+		};
+	},
+	props: {
+		visible: {
+			type: Boolean,
+			default: false
+		},
+		inverted: {
+			type: Boolean,
+			default: false
+		},
+		showCloseButton: {
+			type: Boolean,
+			default: false
+		}
+	},
+	computed: {
+		removeContentBg() {
+			return this.inverted ? 'inverted' : '';
+		}
+	},
+	// Create and set our internal visibility property
+	watch: {
+		// eslint-disable-next-line object-shorthand
+		'visible'() {
```

Comment:
I think if you do this line as just `visible() {` then  you won't need the eslint-disable